### DISCUSSION TITLE
iFrame shows very pixelated model

### DISCUSSION THREAD

**Discussion by piotrszpruta** [google/model-viewer#2823] [Q&A]
### Description

Model in model-viewer on iOS in iFrame is rendering very pixelated even with minimumRenderScale="1". I've checked on Xiaomi Redmi Note 7 and its looking much better.

#### Live Demo
https://d-d-ihelp.github.io/lavello-ar/

### Version
- model-viewer: v1.8.0

### Browser Affected
- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [X] Safari

### OS

- [ ] Android
- [X] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
In fact  you did not set `minimumRenderScale` because it is not an attribute, but a static property. Please follow these directions for setting  static properties: https://modelviewer.dev/examples/loading/#dracoSupport

I've heard some concerning things about iOS throttling 3D severely in iframes. Any info you can give on relative performance between iframe and non would be helpful. I'm afraid when you set the render scale you may end up dropping frame rate significantly.

**Reply by piotrszpruta** [reply]
I did same thing right now like you showed there   https://github.com/google/model-viewer/issues/1708#issuecomment-751896873 but its showing me an error. 
![image](https://user-images.githubusercontent.com/38475451/134970577-1c4cc28c-a256-4f4b-ac10-01bc0063374e.png)

**Reply by piotrszpruta** [reply]
After decreasing vertices to 5k from 25k and setting minimumRenderScale to 1 it still shows exactly same result. But after I move it for few seconds it sometimes loads pretty good model.

**Reply by elalish** [reply]
If it throws an error, then you have not set the property. You'll have  to fix that first to see a  difference. Also, anytime the model stops moving it will revert to full resolution, so it can help to  turn  off `auto-rotate`. It should only decrease resolution to keep the frame rate up.
